client/styles/styles.css

```diff
@@ -405,6 +405,21 @@ body > div:first-of-type > div:first-of-type {
 .col-left {
   padding: 0 15px;
   float: left;
+<<<<<<< HEAD
+}
+
+.col-left-hero {
```

Comment:
I can't find a tag with a class name of `.col-left-hero` anywhere. Which element is it for?

I significantly changed the column structure in my pull request and changing the list column may have significant effect on it. However, I am still debating whether mine should be merged into dev or not. We can discuss tomorrow.